internal/core/api.rs

```diff
@@ -211,9 +211,10 @@ impl WindowSize {
 #[non_exhaustive]
 pub enum GraphicsAPI<'a> {
     /// The rendering is done using OpenGL.
+    #[cfg(feature = "std")]
     NativeOpenGL {
         /// Use this function pointer to obtain access to the OpenGL implementation - similar to `eglGetProcAddress`.
-        get_proc_address: &'a dyn Fn(&str) -> *const core::ffi::c_void,
+        get_proc_address: &'a dyn Fn(&std::ffi::CStr) -> *const core::ffi::c_void,
```

Comment:
why note core::ffi::CStr?  so you don't need the feature flag

Reply:
Ohh, I thought it's in std. Ok, good point :)